assets/scss/local.scss

```diff
@@ -44,3 +44,13 @@
   font-size: inherit;
   line-height: inherit;
 }
+
+.app-u-heading-with-change-link {
```

Comment:
I've added an application style specifically for one of the change links - in the current CIAG UI template that Matt had originally copied it uses a nasty inline style with a `float: right` 😢 
This is a better approach 👍